sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLCLIDriver.scala

```diff
@@ -50,6 +50,7 @@ private[hive] object SparkSQLCLIDriver extends Logging {
   private val prompt = "spark-sql"
   private val continuedPrompt = "".padTo(prompt.length, ' ')
   private var transport: TSocket = _
+  private final val SPARK_HADOOP_PROP_PREFIX = "spark.hadoop."
```

Comment:
`spark.hadoop.` was tribal knowledge and was a sneaky way to stick values into Hadoop `Configuration` object (which can later also pass on to `HiveConf`). What does `spark.hive.` do ? Have never seen such configs and would like to know.

Keeping that aside, are you proposing to drop that prefix at L145 ?